assets/src/js/admin/admin-export.js

```diff
@@ -11,10 +11,77 @@
 
 jQuery.noConflict();
 jQuery( document ).ready( function ( $ ) {
+
+	/**
+	 * Update Exort Donation Form
+	 *
+	 * @since 2.1
+	 */
+	function give_update_donation_form() {
```

Comment:
Prefix with `give_export_update_donation_form()` because this is a very common function name that could cause conflicts

Reply:
Updated functions name